CMakeLists.txt

```diff
@@ -44,6 +44,7 @@ endif()
 find_package(Threads REQUIRED)
 find_package(nlohmann_json "3.6.1" REQUIRED)
 find_package(OpenSSL REQUIRED)
+find_package(fmt "8.1.1")
```

Comment:
Your PR for irods/irods contains the following:
```cmake
if (CMAKE_VERSION VERSION_LESS "3.19.0")
￼  find_package(fmt "8.1.1" REQUIRED)
else()
￼  find_package(fmt 8.1.1...<10.0.0 REQUIRED)
endif()

... <snip> ...

if (TARGET fmt::fmt-header-only AND NOT TARGET fmt::fmt)
￼  add_library(fmt::fmt ALIAS fmt::fmt-header-only)
endif()
```

Is there a reason this project doesn't do that too?

Reply:
As far as I can tell, the problem preventing irods's compatibility with fmt 10+ only affects the main repo.

Reply:
Got it. I assume that situation is on your TODO list to tackle at a later time?

Do we have an issue for that?

Reply:
irods/irods#8454